src/powershell-native/nativemsh/pwrshexe/MshResources.rc

```diff
@@ -1,4 +1,4 @@
-// Copyright (C) 2005 Microsoft Corporation
+// Copyright (c) Microsoft Corporation
```

Comment:
I believe we use common pattern "Copyright (c) Microsoft.  All rights reserved."